### DISCUSSION TITLE
请问如何配置 publicpath？

### DISCUSSION THREAD

**Discussion by LukeLiou** [vuepress/core#1398] [Q&A]
vuepress build docs后，在浏览器访问资源 dist，许多文件 404，想请问如何配置 publicpath？

**Comment by meteorlxy** [comment]
- https://v2.vuepress.vuejs.org/reference/config.html#base
- https://v2.vuepress.vuejs.org/reference/config.html#public

**Reply by LukeLiou** [reply]
使用 vuepress，是直接在根目录新建 vite.config.js，配置public 就能生效吗，我尝试过，发现并不会读取此配置

**Reply by meteorlxy** [reply]
我觉得我们文档写得还算挺详细的，要不要去读一下？

**Reply by LukeLiou** [reply]
嗨，meteorlxy：
其实我的初衷是将 vuepress build 后的 dist 放到 oss 进行访问，但是 vueprss 打包机制，使得浏览器不能直接访问 dist 资源，我已拜读文档，故想改配置，实现oss 部署 dist 的目的，在实操期间未能起效，所以前来请教，盼复
